### DISCUSSION TITLE
Welcome to app-extension-apollo Discussions!

### DISCUSSION THREAD

**Discussion by smolinari** [quasarframework/app-extension-apollo#98] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ejez** [comment]
Thanks for your efforts Scott!

I hope the community will find it useful and share ideas.
